### ISSUE TITLE
IPython prompt character encoding

### ISSUE THREAD

**Issue by ste-goldstein** [wuub/SublimeREPL#570]
When I load an Ipython REPL, the initial messages display fine, but the prompt doesn't seem to be in the correct character encoding.
![2019-04-03_150333](https://user-images.githubusercontent.com/6606412/55459106-51ee6c00-5621-11e9-9b1d-03a3cf5fbd5a.png)

**Comment by uvamerstorfer** [comment]
I have the same issue. Tried some things found on the net, but nothing worked.

**Comment by ste-goldstein** [comment]
@uvamerstorfer Do you have any links of what you tried ? I couldn't find anything related to this. Thanks.

**Comment by zouhairm** [comment]
can confirm have the same issue. @skualos did you ever fix?

**Comment by ste-goldstein** [comment]
@zouhairm no, still have the issue. let me know if you find anything about it.

**Comment by ste-goldstein** [comment]
I replaced the ipy_repl.py file with the one from the link below and encoding is correct now
https://gist.github.com/mcdevitts/25eb4635109fa78920d05658601186a0#file-ipy_repl-py
